app/aws/metrics.py

```diff
@@ -46,19 +48,22 @@ def put_batch_saving_inflight_metric(metrics_logger: MetricsLogger, count: int):
         metrics_logger.set_namespace("NotificationCanadaCa")
         metrics_logger.put_metric("batch_saving_inflight", count, "Count")
         metrics_logger.set_dimensions({"created": "True"})
+        if Config.FF_PRIORITY_LANES:
+            metrics_logger.set_dimensions({"notification_type": queue._suffix})
+            metrics_logger.set_dimensions({"priority": queue._process_type})
         metrics_logger.flush()
     except ClientError as e:
         message = "Error sending CloudWatch Metric: {}".format(e)
         current_app.logger.warning(message)
     return
 
 
-def put_batch_saving_inflight_processed(metrics_logger: MetricsLogger, count: int):
+def put_batch_saving_inflight_processed(metrics_logger: MetricsLogger, queue: RedisQueue, count: int):
```

Comment:
where is the queue being used? Also can we update the args if it is?

Reply:
It's used here to pull out the priority and notification type
https://github.com/cds-snc/notification-api/pull/1539/files/249700e0267bfe31027d00ac13d76c02bb219b58#diff-209447dfcd7d9a8168a8445e2cce4f35ddfdcc3d63a5a307aacc2d791726797dR76-R77